### PR TITLE
Update uk_ua.json

### PR DESCRIPTION
**Pull request by TheFisherGuy**
Idk why did a huge chunk of my work just disappear

Fixes Finished it finally

### CODE REVIEW

File: src/main/resources/assets/changed/lang/uk_ua.json [hunk 1]
```diff
@@ -403,5 +403,111 @@
...
+
+"changed.compact_disc.title.dark_latex_wolf": "Таємниця за маскою",
+"changed.compact_disc.content.dark_latex_wolf": "Як було сказано раніше, тільки тварини здатні пережити лють епідемії. У наших спробах трансфурувати людей ми створили вовкоподібних істот з маскою на обличчі, змішавши наш первинний латекс і чорну фарбу. Рекомендуються подальші випробування.",
+"changed.compact_disc.title.dark_latex_yufeng": "Дракони в масках?"
+"changed.compact_disc.content.dark_latex_yufeng": "Після використання тих же матеріалів, що і у вовку з темного латексу але додавши перо, ми створили істоту здатну планувати повітрям, але все ще не вміючу літати. Воно майже схоже на дракона, але тільки з маскою.",
+"changed.compact_disc.title.dark_latex_dragon": "Дракони"
+
+"changed.compact_disc.content.dark_latex_dragon": "Не дивлячись на те, що ця істота походить від вовків з темного латексу, вона на подив не має маски. Ці ті, що використовують свої задні лапи "Дракони" здатні планувати. Об'єднавши мембрану фантома і те ж за допомогою чого комбінуються вовки з темного латексу, створюється ДНК дракона.",
```
Comment (LtxProgrammer) [inline src/main/resources/assets/changed/lang/uk_ua.json:427]: Syntax error: unescaped quotes

File: src/main/resources/assets/changed/lang/uk_ua.json [hunk 2]
```diff
@@ -403,5 +403,111 @@
...
+"changed:advancements.main.root.title": "Changed",
+"changed:advancements.main.root.description": "Ласкаво просимо в майбутнє",
+
+"changed:advancements.main.transfur.title": "Це назавжди!?",
+"changed:advancements.main.transfur.description": "Трансфурмуйтеся",
+
+"changed:advancements.main.transfur_aquatic.title": "Піду поплаваю в річці глибокій..."
+"changed:advancements.main.transfur_aquatic.description": "Трансфурмуйтеся в будь-який водний латекс.",
+
+"changed:advancements.main.aquatic_swimming.title": "Я шукав "Москву"!",
```
Comment (LtxProgrammer) [inline src/main/resources/assets/changed/lang/uk_ua.json:455]: Syntax error: unescaped quotes

**Review by LtxProgrammer — CHANGES_REQUESTED** [review]
It seems like the entries aren't formatted consistently, where the new entries are not spaced in the beginning of the line.

File: src/main/resources/assets/changed/lang/uk_ua.json (hunk 2 above)
Comment (TheFisherGuy) [reply src/main/resources/assets/changed/lang/uk_ua.json:455]: Ok

**Review by LtxProgrammer — APPROVED** [review]

**Review by TheFisherGuy — COMMENTED** [review]
finally

**Pull request merged**